DEMO/LEDsetup.py

```diff
@@ -1,21 +1,16 @@
-
-# coding: utf-8
-
 # In[1]:
 
-# The lines reset a driver called spidev, 
+# The lines reset a driver called spidev,
 # that is used to drive the signal sent to the lights from the raspberry pi
 from subprocess import call
 call(["sudo", "chmod", "a+rw", "/dev/spidev0.0"])
 
 # Importing some parts of a 'library' of programs called bibliopixel
-# Bibliopixel has the all the functions we use to light up LEDs 
+# Bibliopixel has the all the functions we use to light up LEDs
 # like led.set(), led.fill() etc
 
-from bibliopixel import *
-from bibliopixel.drivers.LPD8806 import *
-from bibliopixel.drivers.driver_base import *
-from bibliopixel import LEDStrip
+from bibliopixel.layout import Strip
```

Comment:
Here I import exactly the symbols I need.  I forgot one - I added it in another commit.